common/djangoapps/third_party_auth/models.py

```diff
@@ -285,6 +285,10 @@ def get_authentication_backend(self):
         """Gets associated Django settings.AUTHENTICATION_BACKEND string."""
         return '{}.{}'.format(self.backend_class.__module__, self.backend_class.__name__)
 
+    def is_auth0(self):
+        """Define if it's an Auth0 provider we are working with."""
```

Comment:
Same with the doc-string.
```suggestion
        """
        Define if it's an Auth0 provider we are working with.
        """
```